File: integrations/aws-strands/error-code-corpus/python/frame_in_accessor.py
"""A frame built inside a property accessor."""

from ag_ui.core import EventType, RunErrorEvent

failure_code = "AT_FILE_SCOPE"


class Reporter:
    @property
    def failure(self) -> RunErrorEvent:
        failure_code = "IN_ACCESSOR"
        return RunErrorEvent(
            type=EventType.RUN_ERROR,
            message="Built in an accessor.",
            code=failure_code,
        )

File: integrations/aws-strands/error-code-corpus/typescript/frame_in_accessor.ts
/** A frame built inside a get accessor. */
import { EventType, type BaseEvent } from "@ag-ui/core";

const failureCode = "AT_FILE_SCOPE";

export class Reporter {
  get failure(): BaseEvent {
    const failureCode = "IN_ACCESSOR";
    return {
      type: EventType.RUN_ERROR,
      message: "Built in an accessor.",
      code: failureCode,
    };
  }
}
